src/houses/KM_Houses.pas

```diff
@@ -1305,7 +1305,7 @@ procedure TKMHouse.UpdateResRequest;
 
 
 procedure TKMHouse.UpdateState;
-//var HouseName: string;
+var HouseNotOccupMsgId: Integer;
```

Comment:
It's better to rename "NotOccup" to "Unoccupied", so that the term used is consistent.